### PR TITLE
Fix linter issues

### PR DESCRIPTION
**Pull request by yukiisbored**
These are fixes for issues reported by coala

I hope you have a very awesome holiday!

Cheers.

www

PS: If you're looking something that can fill the hole for a9a5181, try gitmate.io. It can propose patches for the one who creates the PR.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -145,23 +145,24 @@
...
-                     </p>
-                     <ul>
-                       <li id="name">everyone !</li>
-
-                     </ul>
-                   </div>
-                 </div>
-               </div>
-             </section>
+            <section id="contributors-thanks" href="#contributors">
```
Comment (ankitrgadiya) [inline index.html:148]: Why you removed "Thanks contributors section"?

File: .travis.yml [hunk 2]
```diff
@@ -11,10 +11,7 @@ script:
   - bundle exec jekyll build
   - percy snapshot _site/
   - ruby ./scripts/image_checker.rb
-  - docker run --volume=$(pwd):/app --workdir=/app coala/base:pre /bin/bash -c
-    "coala --ci --apply-patches --no-orig;
-    coala-ci || true"
-  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then bash ./scripts/coala_patch.sh; fi'
```
Comment (ankitrgadiya) [inline .travis.yml:15]: Why are you removing this??

Comment (yukiisbored) [reply .travis.yml:15]: Read the commit message https://github.com/fossasia/gci16.fossasia.org/pull/656/commits/a9a518148de424626d5b96c4c03b5fe4e37668c3

**TL;DR** It doesn't work and It nullifies coala thus making a bad example

File: index.html (hunk 1 above)
Comment (yukiisbored) [reply index.html:148]: I didn't, Read the diffs below. I was fixing indentation and an issue reported by the linters.
Please be careful, I know diffs are a bit hard to read, but try to read it carefully.

Comment (ankitrgadiya) [reply index.html:148]: Okay Sorry my bad @yukiisbored

**Review by ankitrgadiya — APPROVED** [review]

**Comment by yukiisbored** [comment]
@robbyoconnor Sorry to annoy you, but Can you fix @checkbot ?

`index.html` is changed due to the linter errors reported.

`default` section:
```
|    | [NORMAL] SpaceConsistencyBear:
|    | Line contains following spacing inconsistencies:
|----|    | /app/index.html
|    |++++| /app/index.html
| 161| 161|                </div>
| 162| 162|              </section>
| 163| 163| 
| 164|    |- 	     <!-- Contributor Section -->
|    | 164|+         <!-- Contributor Section -->
| 165| 165|             <section id="contributors" href="#contributors" class="contributors-sec">
| 166| 166|                 <div id="contributors-list" class="row">
| 167| 167|                     <!-- Start ignoring BootLintBear -->
```

`html` section:
```
index.html
| 148| ••••••••••••<section•id="contributorsThanks"•href="#contributors">
|    | [MAJOR] HTMLLintBear:
|    | ID and Class Name Delimiters: The id names should be lowercase and with hyphens instead of underscores: Remove the offending characters from the id "contributorsThanks".

index.html
| 149| ••••••••••••••<div•id="contributorsThanks-list"•class="row">
|    | [MAJOR] HTMLLintBear:
|    | ID and Class Name Delimiters: The id names should be lowercase and with hyphens instead of underscores: Remove the offending characters from the id "contributorsThanks-list".

index.html
| 164| •»   •••••<!--•Contributor•Section•-->
|    | [MAJOR] HTMLLintBear:
|    | Indentation: Do not use tabs: Remove the tabs.
index.html
| 150| •••••••••••••••••<div•class="content">
|    | [MAJOR] BootLintBear:
|    | Only columns (`.col-*-*`) may be children of `.row`s
```

**Comment by ankitrgadiya** [comment]
@yukiisbored Check [this](https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/maintainers.md#notes-for-mantainers) as you are changing the index.html file intentionally.

**Comment by yukiisbored** [comment]
@ankitrgadiya What do you mean by "this" ?

**Comment by ankitrgadiya** [comment]
@yukiisbored Oops! my bad again. Forgot to add link. [Here](https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/maintainers.md#notes-for-mantainers)

**Comment by yukiisbored** [comment]
@ankitrgadiya "PR Body" as in the PR message, right?

**Comment by ankitrgadiya** [comment]
Yeah

**Comment by yukiisbored** [comment]
Can I please see the details for `percy`?

**Comment by yukiisbored** [comment]
I don't see any differences with both live and my branch. Can I please see the details for percy or at least a pointer?

**Comment by abishekvashok** [comment]
Wow, not even the maintainers have access! Don't worry @yukiisbored someone like @niccokunzmann or @Princu7 would help you...

File: .travis.yml [hunk 3]
```diff
@@ -13,7 +13,7 @@ script:
   - ruby ./scripts/image_checker.rb
   - docker run --volume=$(pwd):/app --workdir=/app coala/base:pre /bin/bash -c
     "coala --ci --apply-patches --no-orig;
-    coala-ci || true"
+    coala --ci"
```
Comment (robbyoconnor) [inline .travis.yml]: No need for this. `--ci` is already passed on Line 15. You're running coala twice.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]

**Comment by robbyoconnor** [comment]
percy is gonna see a difference...we're changing things...just ignore percy failing...

File: .travis.yml (hunk 3 above)
Comment (jayvdb) [reply .travis.yml]: Wow; it was running twice. Ouch.
So this PR should help solve https://github.com/fossasia/gci16.fossasia.org/issues/226 .

Comment (robbyoconnor) [reply .travis.yml]: @jayvdb it's being run twice still unless line 16 is removed...I added `--ci` I will make this chnage -- back this change out @yukiisbored

Comment (robbyoconnor) [reply .travis.yml]: I merged #665 - which made this change -- @yukiisbored -- to resolve this -- just back this change out..when you'll get a conflict -- just take what's currently in `gh-pages` branch in this repo.

**Review by robbyoconnor — APPROVED** [review]

**Comment by robbyoconnor** [comment]
@yukiisbored -- I resolved the merge conflict for you -- no work needs to be done on your part! 

Are we good to merge this everybody?

Also Happy Holidays!

**Comment by robbyoconnor** [comment]
Merging this -- as now Travis fails if we don't have good code and you fixed issues that are failing.

**Comment by robbyoconnor** [comment]
You broke things -- the contributer segment...I should have checked closer...

**Comment by yukiisbored** [comment]
@robbyoconnor The one that says `[ Thanks to contributors <name> ]` That's already broken to begin with.

**Comment by yukiisbored** [comment]
Because it uses `list` instead of `span`

**Comment by robbyoconnor** [comment]
I'm in favor of nuking it.

On Dec 24, 2016 20:20, "Kaisar Arkhan (Yuki)" <notifications@github.com>
wrote:

@robbyoconnor <https://github.com/robbyoconnor> The one that says [ Thanks
to contributors <name> ] That's already broken to begin with.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/fossasia/gci16.fossasia.org/pull/656#issuecomment-269106464>,
or mute the thread

**Comment by yukiisbored** [comment]
@robbyoconnor Are you serious? Do you really want to remove that piece of text? If so, I'm gonna create an issue for it.

**Comment by robbyoconnor** [comment]
I mean either that or fix it...brownie points for fixing it :D
